### ISSUE TITLE
Anormal buffer duration

### ISSUE THREAD

**Issue by MinDBreaK** [marcelveldt/plugin.audio.spotify#81]
The plugin appear to take a very long time to buffer (sometimes it can take up to 20 to 25 sec) but seems random. 

It doesn't seem to be a network issue as I have no problem when I'm switching to another device.

Anyway you're doing a great job ! 🥂

**Comment by luzik** [comment]
Yeah ..same here. When I am using spotifyd (also librespot based) a feel after changing song  is quite realtime
